packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -486,6 +486,13 @@ export class LoadTestDataStoreModel {
 			);
 		}
 	}
+
+	private isPermanentlyDisconnected(): boolean {
+		return (
+			this.runtime.disposed ||
+			(!this.runtime.connected && this.runtime.deltaManager.readOnlyInfo.readonly === true)
```

Comment:
This does not feel right - feels like poor-man's implementation of "closed" state that should be available to container runtime, no?